Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,95 +1,143 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
-        /// <summary>
-        /// Sort the array in ascending order.
-        /// If array empty or null- don't do anything.
-        /// </summary>
-        /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (IsNullArray(array))
```

Comment:
IsNullOrEmpty

Reply:
Otherwise you could just ``(array is null)`` :D Which is literally less code and no extra method 😄